### PR TITLE
css url() and url src options

### PR DESCRIPTION
**Pull request by smol-honk**
Added url option for svgr. And updated to handle css url imports.

To import as a src in jsx or tsx:
```
import yourSvgUrl from './mySvg.svg?url';
```

It should automatically work for css imports as well.

For the package-lock.json update, it was to fix vulnerable dependencies with `npm audit fix`.

Related issues #10 

This is my first time working with esbuild plugins, so let me know if I did something wrong!

### CODE REVIEW

**Comment by kazijawad** [comment]
Thank you for the PR! A minor nit — it looks like your changes also changed the formatting. Would you be able to revert the code formatting changes? This would make it easier to understand the diff.

**Comment by smol-honk** [comment]
@kazijawad Totally can do! Sorry about the formatting! Must be my prettier coming into play. Appreciate you looking at this 😄

File: index.js [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
     setup(build) {
         const svgr = require('@svgr/core').transform;
         const fs = require('fs');
+        build.onResolve({ filter: /\.svg$/ }, async (args) => {
+            switch (args.kind) {
+                case 'import-statement':
+                case 'require-call':
+                case 'dynamic-import':
+                case 'require-resolve':
+                    return;
```
Comment (smol-honk) [inline index.js:14]: follow default return behavior

File: index.js [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
 
         build.onLoad({ filter: /\.svg$/ }, async (args) => {
             const svg = await fs.promises.readFile(args.path, 'utf8');
-            const contents = await svgr(svg, { ...options }, { filePath: args.path });
+            const contents = await svgr(
+                svg,
+                { ...options },
+                { filePath: args.path }
+            );
+            if (args.suffix === '?url') {
```
Comment (smol-honk) [inline index.js:29]: This way we can import the asset svg as well similar to: https://react-svgr.com/docs/webpack/#use-svgr-and-asset-svg-in-the-same-project

**Comment by kazijawad** [comment]
Thank you for making those changes, this all looks good to me. I'm planning on testing these changes in a project this weekend and then will merge this.

**Comment by kazijawad** [comment]
Ok this looks good to me, I'll go ahead and merge this and release a minor version.

**Pull request merged**